coach/src/concept-coach/index.cjsx

```diff
@@ -101,7 +101,7 @@ class ConceptCoachAPI extends EventEmitter2
     @removeAllListeners()
 
   remove: ->
```

Comment:
Both `initialize()` and `destroy()` call this function, but I'm guessing each one happens only once so it's fine?